tests/Unit/ApplicationWatchPathsTest.php

```
@@ -0,0 +1,368 @@
+/**
+ * Use the shared implementation from Application model
+ */
+function matchWatchPaths(array $changed_files, ?array $watch_paths): array
+{
+    $modifiedFiles = collect($changed_files);
+    $watchPaths = is_null($watch_paths) ? null : collect($watch_paths);
+
+    $result = Application::matchPaths($modifiedFiles, $watchPaths);
+
+    return $result->toArray();
+}
```

PHPMD (2.15.0)

The parameter $changed_files is not named in camelCase. (undefined)

(CamelCaseParameterName)